src/bootstrap/config.rs

```diff
@@ -516,6 +517,7 @@ struct Rust {
     new_symbol_mangling: Option<bool>,
     profile_generate: Option<String>,
     profile_use: Option<String>,
+    profile_sample_use: Option<String>,
```

Comment:
The changes to this file should no longer be necessary, as the rest of the bootstrap support was dropped.